src/iosrv/srv.c

```diff
@@ -275,6 +364,7 @@ dss_sched_free(ABT_sched sched)
 {
 	struct sched_data *p_data;
 
+	ds_check_expire_list(true);
```

Comment:
I think it's too late to do force wakeup here, current progress ULT will wait all ULTs exited before exiting, so the timeout ULT list should already be empty when reaching here.

According to current implementation, this forcing wakeup should be done in the progress ULT before waiting other ULTs exiting, right?

Reply:
ah, ok